spec/controllers/concerns/idv/phone_otp_rate_limitable_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Idv::PhoneOtpRateLimitable, type: :controller do
```

Comment:
thanks for adding specs for this previously untested controller!!!

Reply:
+1 to that!